code/__HELPERS/events.dm

```diff
@@ -0,0 +1,52 @@
+#define UNLIT_AREA_BRIGHTNESS 0.2
+
+/**
+ * Finds us a generic maintenance spawn location.
+ *
+ * Goes through the list of the generic mainteance landmark locations, checking for atmos safety if required, and returns
+ * a valid turf. Returns MAP_ERROR if no valid locations are present.
+ * Returns nothing and alerts admins if no valid points are found. Keep this in mind
+ * when using this helper.
+ */
+
+/proc/find_maintenance_spawn(atmos_sensitive = FALSE, require_darkness = FALSE)
+	var/list/possible_spawns = list()
+	for(var/spawn_location in GLOB.generic_maintenance_landmarks)
+		var/turf/spawn_turf = get_turf(spawn_location)
+
+		if(atmos_sensitive && !is_safe_turf(spawn_turf))
+			continue
+
+		if(require_darkness && spawn_turf.get_lumcount() > UNLIT_AREA_BRIGHTNESS)
+			continue
+
+		possible_spawns += spawn_turf
+
+	if(!length(possible_spawns))
+		message_admins("No valid generic_maintenance_landmark landmarks found, aborting...")
+		return
```

Comment:
```suggestion
		return null
```

We should do it like this for explicitness and reap benefits elsewhere in readability (null was already being returned implicitly, but returning it explicitly will make my other reviews better)